features/step_definitions/legal_services_stepdef.rb

```diff
@@ -0,0 +1,89 @@
+Given("I am on the legal services start page") do
```

Comment:
A lot of the step definitions here will be common between services so we should consider moving them into the `common_stepdef.rb` file